src/styles/blocks/header.scss

```diff
@@ -0,0 +1,71 @@
+.header {
+  background-size: cover;
+  background-position: center;
+  background-image: url("/image/image31.png");
+
+  @include onTablet {
+    background-image: url("/image/image3.png");
+  }
+
+  @include onDesktop {
+    background-image: url("/image/header.png");
+  }
+
+  &__content {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  &__title {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 100%;
+    text-align: center;
+    letter-spacing: -1.5px;
+    color: #fff;
+    margin: auto;
+
+    @include onTablet {
+      font-size: 64px;
+    }
+
+    @include onDesktop {
+      font-size: 80px;
+    }
+  }
+
+  &__call {
+    margin-right: 30px;
+    text-decoration: none;
+
+    @include onDesktop {
+      &::before {
+        content: "The Met Fifth Avenue +1 212-535-7710"
+          "                     The Met Cloisters    +1 212-923-3700";
+        position: relative;
+        white-space: pre-wrap;
+        top: 80%;
+        transform: translate(-100%, -50%);
+        display: block;
+        width: 201px;
+
+        font-weight: 700;
+        font-size: 16px;
+        color: #fff;
```

Comment:
Use a variable for the color #fff, to make it easier to update the color scheme in the future.